### ISSUE TITLE
Suggestion: Copy and Pasting barrel Configurations

### ISSUE THREAD

**Issue by thebrianlenz** [LatvianModder/YABBA#118]
Configuring multiple barrels with identical layouts can be extremely tedious.

Perhaps adding another functionality of the Barrel Wrench could make this less cumbersome by "copying" and "pasting" configurations quickly to barrels (similar to the Conduit Probe of EnderIO).

**Comment by DizzyDJW** [comment]
Any news on this yet? Would love to see not just this but also small icons when looking at a barrel that shows whether it has an upgrade or not.
I think it'd be easier to just add a "Lock All" button to the Barrel Controller block.

**Comment by Ralsch** [comment]
At least you can Shift-RClick the controller and go through all barrels to configure them (1.12). Has visuals to show locked status. Better than finding all barrels in the wall. ^^

**Comment by DizzyDJW** [comment]
True, but as soon as Storage Drawers has a default recipe for the Creative/Infinite Storage Upgrade it will be the better storage mod. You can already lock all AND the best part, something that Yabba is still missing is the ability for the locked drawers to not allow anything to be automatically placed into them until you manually place the first object.
I feel like Yabba is still missing too many crucial features, and the only reason I use it is for the infinite upgrade. I do enjoy the unique ability to alter the shape and size of the barrels tho.
